website/config/routes.js

```diff
@@ -492,8 +492,8 @@ module.exports.routes = {
 
   // Fleet UI
   // =============================================================================================================
-  // These are external links not maintained by Fleet. We can point the Fleet UI to redirects here instead of the
-  // original sources to help avoid broken links.
+  // Redirects for external links from the Fleet UI & CLI, including to fleetdm.com and to external websites not 
```

Comment:
```suggestion
  // Redirects for external links from the Fleet UI & CLI, including to fleetdm.com and to external websites not
```
This trailing whitespace is causing the website test to fail.